MSAL/src/native_auth/public/MSALNativeAuthUserAccountResult.swift

```diff
@@ -75,25 +75,41 @@ import Foundation
     ///   - forceRefresh: Ignore any existing access token in the cache and force MSAL to get a new access token from the service.
```

Comment:
can you please align with MSAL Android comment for this method?

Reply:
Done.